src/_guides/language/language-tour.md

```diff
@@ -4241,6 +4240,33 @@ Keep in mind the following when you use deferred loading:
   using <code>deferred as <em>namespace</em></code>.
   The `loadLibrary()` function returns a [`Future`](/guides/libraries/library-tour#future).
 
+#### The `library` directive
```

Comment:
Let's make a simpler this have a simpler anchor:

```suggestion
#### The `library` directive {#library-directive}
```

Reply:
Getting this from `tool/check-links`:
```
http://localhost:5500/guides/language/language-tour
- (5288:20) 'library' => http://localhost:5500/guides/language/language-tour#the-library-directive 
(HTTP 200 but missing anchor)

```